### PR TITLE
Add Initial Support for PoliCheck and CredScan

### PR DESCRIPTION
**Pull request by CartBlanche**

### CODE REVIEW

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -112,3 +121,17 @@ stages:
               pr_number: $(pull_request_number)
               merge_method: 'merge'
             displayName: Merge PR# $(System.PullRequest.PullRequestNumber)
+
+  - stage: PoliCheck
```
Comment (mcumming) [inline azure-pipelines.yaml:125]: For consistency with what I have for the designer repo, call the stage code_analysis and a display name of "Code Analysis" and move the stage to the top of the file.

File: azure-pipelines.yaml [hunk 2]
```diff
@@ -112,3 +121,17 @@ stages:
...
+    dependsOn: ['Windows', 'Mac']
```
Comment (mcumming) [inline azure-pipelines.yaml:126]: This is not needed and will just extend the time to completion

File: azure-pipelines.yaml [hunk 3]
```diff
@@ -112,3 +121,17 @@ stages:
...
+    displayName: PoliCheck
+    jobs:
+    # Check - "PoliCheck "
+    - job: run_poli_check
```
Comment (mcumming) [inline azure-pipelines.yaml:130]: for consistency, use run_polichack_compliance with a display name of "Policheck Compliance"

File: azure-pipelines.yaml (hunk 2 above)
Comment (CartBlanche) [reply azure-pipelines.yaml]: Done.

File: azure-pipelines.yaml (hunk 1 above)
Comment (CartBlanche) [reply azure-pipelines.yaml]: Done.

File: azure-pipelines.yaml (hunk 3 above)
Comment (CartBlanche) [reply azure-pipelines.yaml]: Done.

**Comment by CartBlanche** [comment]
@mcumming Added your suggestions, including CredScan.

**Pull request merged**